Removing dependency on embedded convenience copy of libtinyxml.Index: presage-0.8.3/src/lib/core/profile.h
===================================================================
Index: presage-0.8.3/src/lib/core/profile.h
===================================================================
--- presage-0.8.3.orig/src/lib/core/profile.h	2010-06-19 19:20:54.000000000 +0100
+++ presage-0.8.3/src/lib/core/profile.h	2010-06-19 19:21:23.000000000 +0100
@@ -27,7 +27,7 @@
 
 #include "config.h"
 
-#include "tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 #include <vector>
 #include <string>
Index: presage-0.8.3/src/lib/core/profileManager.h
===================================================================
--- presage-0.8.3.orig/src/lib/core/profileManager.h	2010-06-19 19:20:50.000000000 +0100
+++ presage-0.8.3/src/lib/core/profileManager.h	2010-06-19 19:21:46.000000000 +0100
@@ -27,7 +27,6 @@
 
 #include "profile.h"
 #include "defaultProfile.h"
-#include "../tinyxml/tinyxml.h"
 #include "combiner.h"
 #include "logger.h"
 #include "dispatcher.h"
